.drone.yml

```diff
@@ -3,13 +3,16 @@ kind: pipeline
 name: default
 type: kubernetes
 
+environment:
+  IMAGE_URL: quay.io/ukhomeofficedigital
+  IMAGE_REPO: file-vault
+
 steps:
 - name: build
   pull: if-not-exists
-  image: node:14
+  image: node:18
   commands:
   - yarn install --frozen-lockfile
-  - yarn run test
```

Comment:
the tests no longer work?

Reply:
It works for me in local

Reply:
Cool, why is this taken out of the pipeline?